AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -38,6 +38,7 @@ andoveradvertiser.co.uk,asianimage.co.uk,banburycake.co.uk,barryanddistrictnews.
 escortbook.com,looper.com,fromdoctopdf.com,learnersdictionary.com,spanishdict.com,merriam-webster.com,digi24.ro,antena3.ro,wikihow.com.tr,wikihow.cz,wikihow.it,wikihow.jp,wikihow.vn,wikihow.com,mobil.se,digisport.ro,periscope-solutions.com###gdpr
 !
 castbox.fm##.cookies-conf
+unimelb.edu.au##.tealium_privacy_prompt
```

Comment:
Not shown in code for me.

Reply:
<details><summary>Screenshot:</summary>

![cookie](https://user-images.githubusercontent.com/33488229/64707508-c6c0ae80-d4bb-11e9-88a1-82c82f89607c.png)
</details><br/>